### DISCUSSION TITLE
What units are used in the console log?

### DISCUSSION THREAD

**Discussion by andras-ballai** [ConsenSys-archive/ganache#4406] [General]
When using ganache I get the following transaction log:
```
eth_sendRawTransaction
  Transaction: ...
  Contract created: ...
  Gas usage: 2894024
  Block number: 47
  Block time: ...
```
I'd like to know what the unit of "Gas usage" is here, I'm assuming Gwei, but I'd like to be certain.  
  
I'd also appreciate if the unit was always provided both in the CLI --help command and in outputs.

**Comment by jeffsmale90** [comment]
Hi, thanks for raising this - the units are "Ethereum Gas Units" as described here: [https://ethereum.org/en/developers/docs/gas](https://ethereum.org/en/developers/docs/gas).

We don't output the actual Eth cost for the transaction, the reason being while the Gas Units doesn't vary (for a given transaction), but the actual Eth cost will change drastically, so the Eth cost in the development environment isn't especially meaningful. I'd be interested to understand if providing this information would be useful for your workflow, how we improve this.

**Reply by andras-ballai** [reply]
Absolutely makes sense to notate it in Gas Units, thank you for the clarification.
